doc/developer-guide/developer-guide.rst

```diff
@@ -0,0 +1,71 @@
+.. _Developer_Guide:
+
+###################
+``ts_imsim`` Developer Guide
```

Comment:
I think you can use "Developer Guide", only. Otherwise, it looks like this:

<img width="686" alt="Screenshot 2023-11-07 at 9 06 02 PM" src="https://github.com/lsst-ts/ts_imsim/assets/11474902/a38fa00b-f759-4038-979e-0440069479dd">